### PR TITLE
extracting the css to a separate file so it works for server side rendering

### PR DESCRIPTION
**Pull request by alejandromaka**

### CODE REVIEW

**Comment by PeteJodo** [comment]
I had unpublished the broken `v0.2.2` so there's 3 commits that are currently unpublished that I think should just be reverted in it's entirety since it's what caused the issues. Sorry this probably affects the work you did on top of it.

**Comment by PeteJodo** [comment]
so effectively, we would release a new `v0.2.2` rather then just skipping it and going to `v0.2.3`. Not sure what others think (this would require removing the current `v0.2.2` git tag)

**Comment by hellatan** [comment]
you would have to skip 0.2.2 regardless if it's been unpublished. 0.2.3 is fine

**Comment by kostasmanionis** [comment]
Isn't this a breaking change? i.e. shouldn't it be a major?

**Comment by alejandromaka** [comment]
yeah this is def breaking. maybe 0.3.0? if we wanna keep the 0.

**Comment by hellatan** [comment]
semver says < 1.* means there can be breaking changes until it hits its first major. I'd say 0.3.0 in this case then makes more sense.

**Comment by PeteJodo** [comment]
I was just more concerned with reverting the commits that broke it in the first place but yeah I agree with `v0.3.0` since this is breaking

**Comment by alejandromaka** [comment]
fixed

File: webpack.build.js [hunk 1]
```diff
@@ -25,8 +26,9 @@ module.exports = {
       },
       {
         test: /\.css$/,
-        loader: "isomorphic-style-loader!css-loader!autoprefixer-loader"
+        loader: ExtractTextPlugin.extract('css?localIdentName=[name]__[local]___[hash:base64:5]!autoprefixer-loader')
```
Comment (hellatan) [inline webpack.build.js]: so same thing here - dibs-webpack-scss?

**Comment by alejandromaka** [comment]
bump?

**Comment by robrichard** [comment]
LGTM

**Comment by hellatan** [comment]
do we need two different webpack files?

**Comment by PeteJodo** [comment]
I can't answer if we still need one since I wrote 2 when I was early on in learning webpack but one is for running the examples and the other is for building release

**Comment by hellatan** [comment]
> I can't answer if we still need one since I wrote them when I was early on in learning webpack but one is for running the examples and the other is for building release

I won't block on this but we should try to get this more inlined with our updated scaffolding if possible.

**Comment by kostasmanionis** [comment]
> I won't block on this but we should try to get this more inlined with our updated scaffolding if possible.

Ditch webpack for packages, use rollup! 😄

**Pull request merged**